pepper-apis/src/main/java/org/broadinstitute/ddp/util/LiquibaseUtil.java

```diff
@@ -90,6 +92,11 @@ public static void runChangeLog(String dbUrl, String changeLogFile) {
         }
     }
 
+    public static void releaseResources() {
+        ServiceLocator.reset();
+        DatabaseFactory.reset();
```

Comment:
It looks like `reset()` will set those to new instances. Will it be better and safe to do the following?
```
ServiceLocator.setInstance(null);
DatabaseFactory.setInstance(null);
```

Reply:
I think you are right! DatabaseFactory constructor doesn't do too much but ServiceLocator constructor could read a whole bunch of stuff.